### PR TITLE
Upgraded the Nix install to work with Astal

### PR DESCRIPTION
**Pull request by ThomasCarey4**
You forgot to change the nix installation for Astal lol, was completely broken.

I cleaned up the flake.nix to use the inbuilt ags bundler (instead of bun) and had to modify a few ts imports as the bundler gets confused by the same file being imported by different paths.

### CODE REVIEW

**Comment by ThomasCarey4** [comment]
Whoops, must of missed them thanks!

**Comment by Jas-SinghFSU** [comment]
The file path issue seems to be nix specific.

**Comment by Jas-SinghFSU** [comment]
We should figure out if there is a way to handle that properly first.

**Comment by Jas-SinghFSU** [comment]
Actually I think the import updates are fine but the ts config path resolution seems redundant. That shouldn't be needed?

**Comment by ThomasCarey4** [comment]
> Actually I think the import updates are fine but the ts config path resolution seems redundant. That shouldn't be needed?

yeah ur right the ts config change does nothing lol. Never used typescript so I was just asking chatgpt for it all :)

File: tsconfig.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "jsx": "react-jsx",
         "jsxImportSource": "/usr/share/astal/gjs/gtk3",
         "paths": {
+            "src/*": ["src/*"],
```
Comment (Jas-SinghFSU) [inline tsconfig.json:24]:
```suggestion
```

**Pull request merged**